static/assets/css/main.css

```diff
@@ -71,3 +71,18 @@ blockquote li,
 blockquote::before {
   border-color: #d3d3d3; /* $color-mediumgray */
 }
+
+.env-commandline .env-specifier,
+.env-dashboard .env-specifier,
+.env-commandline .env-specifier a
+.env-dashboard .env-specifier a
+{
+  color: #333; /* $color-textblack */
+}
+
+.env-commandline::before{
+  border-color: #333;
```

Comment:
@rememberlenny Are the color comments autogenerated? I hope so, but if not, maybe add another `/* $color-textblack */` here for consistency.